src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpHttpTraceExportClient.cs

```diff
@@ -28,6 +28,26 @@ protected override HttpContent CreateHttpContent(OtlpCollector.ExportTraceServic
         return new ExportRequestContent(exportRequest);
     }
 
+    private static HttpClient ModifyHttpClient(OtlpExporterOptions options, HttpClient httpClient)
```

Comment:
We can't just throw away the `HttpClient` and make a new one. Users are able to configure SSL/TLS today using factory:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/5dff99f8a0b26ff75248d5f2e478b9c3c42f5678/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs#L134

This change will break any user doing that or doing anything else to the HttpClient they are intending to use here.

Reply:
I modified a little bit, does it look fine? @CodeBlanch

Reply:
If the design looks ok, I will start to write tests :)